test/prefer-key-over-key-code.js

```diff
@@ -0,0 +1,519 @@
+			f.addEventList('some', e => {
+				const {charCode} = e;
+				console.log(event.key)
+			})
+		})`,
+		`foo.addEventListener('click', e => {
+			{
+				const e = {};
+				console.log(e.keyCode);
+			}
+		});`
+	],
+
+	invalid: [
+		{
+			code: `
+				window.addEventListener('click', e => {
+					console.log(e.keyCode);
+				})
+			`,
+			errors: [error('keyCode')]
+		},
+		{
+			code: `
+				foo123.addEventListener('click', event => {
+					if (event.keyCode === 27) {
+					}
+				});
+			`,
+			output: `
+				foo123.addEventListener('click', event => {
+					if (event.key === 'Escape') {
+					}
+				});
+			`,
+			errors: [error('keyCode')]
+		},
+		{
+			code: `
+				foo.addEventListener('click', a => {
+					if (a.keyCode === 27) {
+					}
+				});
+			`,
+			output: `
+				foo.addEventListener('click', a => {
+					if (a.key === 'Escape') {
+					}
+				});
+			`,
+			errors: [error('keyCode')]
+		},
+		{
+			code: `
+				foo.addEventListener('click', (a, b, c) => {
+					if (a.keyCode === 27) {
+					}
+				});
+			`,
+			output: `
+				foo.addEventListener('click', (a, b, c) => {
+					if (a.key === 'Escape') {
+					}
+				});
+			`,
+			errors: [error('keyCode')]
+		},
+		{
+			code: `
+				foo.addEventListener('click', function(a, b, c) {
+					if (a.keyCode === 27) {
+					}
+				});
+			`,
+			output: `
+				foo.addEventListener('click', function(a, b, c) {
+					if (a.key === 'Escape') {
+					}
+				});
+			`,
+			errors: [error('keyCode')]
+		},
+		{
+			code: `
+				foo.addEventListener('click', function(b) {
+					if (b.keyCode === 27) {
+					}
+				});
+			`,
+			output: `
+				foo.addEventListener('click', function(b) {
+					if (b.key === 'Escape') {
+					}
+				});
+			`,
+			errors: [error('keyCode')]
+		},
+		{
+			code: `
+				foo.addEventListener('click', e => {
+					const {keyCode, a, b} = e;
+				});
+			`,
+			errors: [error('keyCode')]
+		},
+		{
+			code: `
+				foo.addEventListener('click', e => {
+					const {a: keyCode, a, b} = e;
+				});
+			`,
+			errors: [error('keyCode')]
+		},
+		{
+			code: `
+				add.addEventListener('keyup', event => {
+					f.addEventList('some', e => {
+						const {keyCode} = event;
+						console.log(event.key)
+					})
+				})
+			`,
+			errors: [error('keyCode')]
+		},
+		{
+			code: `
+				window.addEventListener('click', e => {
+					console.log(e.charCode);
+				})
+			`,
+			errors: [error('charCode')]
+		},
+		{
+			code: `
+				foo11111111.addEventListener('click', event => {
+					if (event.charCode === 27) {
+					}
+				});
+			`,
+			output: `
+				foo11111111.addEventListener('click', event => {
+					if (event.key === 'Escape') {
+					}
+				});
+			`,
+			errors: [error('charCode')]
+		},
+		{
+			code: `
+				foo.addEventListener('click', a => {
+					if (a.charCode === 27) {
+					}
+				});
+			`,
+			errors: [error('charCode')],
+			output: `
+				foo.addEventListener('click', a => {
+					if (a.key === 'Escape') {
```

Comment:
Add tests for more keys than just Escape. I'd like to see tests for the following `keyCode`'s:

- 13
- 38
- 190
- 221

Reply:
Done!